### PR TITLE
New creation modal for event subscriptions

### PR DESCRIPTION
**Pull request by mrCherry97**
**Description**
We provide a new creation modal for event subscriptions.

Changes proposed in this pull request:

- New create view with simple and advanced forms,
- New view and notification for creating a subscription in the function and service views
- Added tests for:
   - Create,
   - Displayed information,
   - Update,
   - Delete,
- Modified tests for creating a subscription in the function view,
- Added test for adding an application.
- Add tooltips and translations

**Related issue(s)**
Closes: #701 
Closes: [Kyma Issue](https://github.tools.sap/kyma/backlog/issues/2150)

### CODE REVIEW

File: tests/tests/in-cluster-eventing.spec.js [hunk 1]
```diff
@@ -172,7 +192,39 @@ context('In-cluster eventing', () => {
...
+    cy.getIframeBody()
+      .contains('Advanced')
+      .click();
+
+    cy.getIframeBody()
+      .find('[placeholder="Event type"]')
+      .clear()
+      .type('sap.kyma.custom.test-mock-app.order.canceled.v2');
+
+    cy.wait(4000);
```
Comment (mrCherry97) [inline tests/tests/in-cluster-eventing.spec.js:217]: I have to delete it, have to do it later.

**Review by akucharska — CHANGES_REQUESTED** [review]
We should not mark missing fields with red outline on form opening. It should be done when we loose focus on the input
Screen Shot 2021-12-29 at 08 06 18

**Review by akucharska — DISMISSED** [review]

File: core/src/i18n/en.yaml [hunk 2]
```diff
@@ -781,45 +781,44 @@ domains:
...
-    server: Server error. Contact your cluster admin.
-    subscriptions-not-found: There are no Event Subscriptions yet.
-  filters:
-    event-source: Event Source
-    event-type: Event Type
-    property: Property
-    title: Filters
-    type: Type
-    value: Value
+    empty-event-types: You must provide at least one Event Type.
```
Comment (klaudiagrz) [inline core/src/i18n/en.yaml:796]:
```suggestion
    empty-event-types: You must provide at least one Event type.
```

**Review by klaudiagrz — DISMISSED** [review]

**Comment by mrCherry97** [comment]
"So, for now unfortunately it is 7 segments constraint, we try to fix that soonish, it’s already in our backlog" Info from Tunas, we have to discuss about solution of `Event type`

File: core/src/i18n/en.yaml [hunk 3]
```diff
@@ -787,45 +787,44 @@ domains:
...
+      title: Filters
+      type: Type
+      value: Value
   name_singular: Event Subscription
   sink: Sink
   title: Event Subscriptions
   tooltips:
-    ready: Subscription is active.
+    application-name: Choose an Application from the cluster.
+    event-name: Provide a name for your Event.
```
Comment (klaudiagrz) [inline core/src/i18n/en.yaml:822]:
```suggestion
    event-name: Provide a name for your Event. The name must consist of at least two parts divided by a dot.
```

**Review by klaudiagrz — APPROVED** [review]

**Review by akucharska — APPROVED** [review]

**Pull request merged**